### PR TITLE
small Awwvision updates and cleanup

### PR DESCRIPTION
**Pull request by amygdala**

### CODE REVIEW

**Comment by amygdala** [comment]
Jon, feel free to reassign to someone else.  :) Just minor updates though.

**Comment by amygdala** [comment]
Andrew, Jon is out today, and suggested I reassign to you.

File: python/awwvision/README.md [hunk 1]
```diff
@@ -35,6 +38,10 @@ This example uses [Container Engine](https://cloud.google.com/container-engine/)
             --num-nodes 2 \
             --scopes cloud-platform
 
+    You may need to set your zone first, e.g.:
```
Comment (andrewsg) [inline python/awwvision/README.md]: I think "may" will be confusing for people; how do they know whether they need to set their zone? Can we be more specific about the situations in which people will need to set their zone, or if needed what error message they'll see if they don't?

Comment (amygdala) [reply python/awwvision/README.md]: makes sense-- I just went ahead and told them to do it first.

**Comment by amygdala** [comment]
thank you!  I'll go ahead and pull it in.

**Pull request merged**